COMPARE.md

```diff
@@ -0,0 +1,54 @@
+# Comparing Standard to X
+
+_Where appropriate, we compare with `divnix/paisano`, instead_.
+
+## Comparison with tools in the Nix ecosystem
+
+### flake-utils
+
+`numtide/flake-utils` is a small & lightweight utility with a focus on generating flake file _outputs_ in accordance with the packaging use case built into the Nix CLI tooling.
+
+Paisano, in turn, is an importer with a focus on _code organization_; it still plugs well into a `flake.nix` file, but also preserves its index function by keeping it clean.
+While you _can_ use it to match the schema that Nix CLI expects, it also enables more flexibility as it is not specially optimized for the Nix _package manager use case_.
+
+### flake-parts
+
+`hercules-ci/flake-parts` is a component aggregator with a focus on a flake schema that is built into the Nix CLI tooling that makes use of the NixOS module system for composability and aggregation.
+
+Paisano, in turn, is an importer with a focus on _code organization_; it still plugs well into a `flake.nix` file, but also preserves its index function by keeping it clean.
+While you _can_ use it to match the schema that Nix CLI expects, it also enables more flexibility as it is not specially optimized for the Nix _package manager use case_.
```

Comment:
Isn't this just a copy paste of the above statement?

Reply:
I figured each comparison needs to stand individually, which may entail repetition of the same argument. I diversified it a bit so that it's not as obvious to the eyes.